src/main/java/chess/view/command/Command.java

```diff
@@ -0,0 +1,44 @@
+package chess.view.command;
+
+import java.util.Arrays;
+
+public enum Command {
+
+    START("start", "start"),
+    MOVE("move", "^move [a-h][1-8] [a-h][1-8]$"),
+    END("end", "end")
+    ;
+
+    private final String message;
+    private final String pattern;
```

Comment:
Pattern 컴파일을 미리 해줘서 비용을 줄일 수 있겠네요
```suggestion
    private final Pattern pattern;
```